test/server/cards/03_TWI/units/PloKoonKohtoyah.spec.ts

```diff
@@ -0,0 +1,44 @@
+describe('Plo Koon Kohtoyah', function () {
+    integration(function (contextRef) {
+        describe('Plo Koon Kohtoyah\'s ability', function () {
+            it('should ambush attack for 6', function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['plo-koon#kohtoyah'],
+                        groundArena: ['obiwan-kenobi#following-fate', 'luke-skywalker#jedi-knight'],
+                    },
+                    player2: {
+                        groundArena: ['wampa'],
```

Comment:
can you use a 7 hp unit to test the exact value of raid (I'm using consular security force for this)